_posts/2022-12-05-neptune-ai-integration-released.md

```diff
@@ -0,0 +1,172 @@
+phase to track and visualize experiment results. Neptune can also
+track and visualize the results produced by your
+ZenML pipeline runs, as you make the transition towards a more production
+oriented workflow.
+
+A big shout-out to [@AleksanderWWW](https://github.com/AleksanderWWW), from the Neptune team
+who spearheaded the [Neptune integration](https://zenml.io/integrations/neptune)
+with [#1082](https://github.com/zenml-io/zenml/pull/1082).
+
+## :couple_with_heart_man_man: How Neptune fits with ZenML
+
+One of the coolest things about Neptune is how focused it is on what it does well:
+Being a kick-ass experiment tracker and model registry. Given ZenML does not have a
+model registry abstracton ([yet](https://zenml.io/roadmap)), we decided to focus on
+the experiment tracking part this time around, with the ZenML
+[experiment tracker](https://docs.zenml.io/component-gallery/experiment-trackers)
+abstraction.
+
+Experiment trackers in ZenML let you track your ML experiments by logging extended information about your models,
+datasets, metrics and other parameters and allowing you to browse them, visualize them and compare them between runs.
+In the ZenML world, every pipeline run is considered an experiment, and ZenML facilitates the storage of experiment
+results through Experiment Tracker stack components. This establishes a clear link between pipeline runs and experiments.
+
+## 🤾 Challenges faced while integration
+
+The tougest challengewith the entire story was to try to maintain the state of the Neptune
+run across the lifetime of a pipeline (i.e across steps). Aleksander and the Neptune team solved this with the
+[RunProvider](https://github.com/zenml-io/zenml/blob/main/src/zenml/integrations/neptune/experiment_trackers/run_state.py#L33)
+class that is a singleton object storing and persisting the Neptune run state across
+the pipeline.
+
+## 🪐 Using Neptune as an Experiment Tracker
+
+With the new Neptune integration, one can easily create a Neptune experiment tracker component and use it in any step in
+a pipeline.
+
+A concrete example with a pipeline can be viewed in the [example](https://github.com/zenml-io/zenml/tree/main/examples/neptune_tracking)
+on GitHub. Here is a brief summary:
+
+### 🥞 Create the stack with the Neptune experiment tracker component
+
+Ceate a new `StackComponent`,  and  subsequently a new `Stack` with the type `neptune`. The Neptune tracker stack component has the following options:
+
+* `api_token`: [API key token](https://docs.neptune.ai/setup/setting_api_token) of your Neptune account. If left blank, Neptune will
+attempt to retrieve it from your environment variables.
+* `project`: The name of the project where you're sending the new run, in the form "workspace-name/project-name".
+If the project is not specified, Neptune will attempt to retrieve it from your environment variables.
+
+Note that project_name and entity are optional in the below command:
```

Comment:
Do you mean project_name and token?

Or maybe `--project` and `--api_token` arguments are optional?